File: single-language-analysis/dataset-files/java/TestSmell152.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell152 {

    @Test
    public void testPR655() {
        Hardware pr655 = Hardware.findPhotometer("PR655");
        if (pr655 == null) {
            System.out.println("no device found");
        } else {
            System.out.println("type: " + pr655.getType());
            System.out.println("SN: " + pr655.getDeviceSN());
            if (pr655.getType().equals("D")) {
                return; // Skip test
            }
            pr655.measure();
            System.out.println("lum: " + pr655.getLastLum());
            System.out.println("uv: " + pr655.getLastUV());
            System.out.println("xy: " + pr655.getLastXY());
            System.out.println("tristim: " + pr655.getLastTristim());
            Object[] lastSpectrum = pr655.getLastSpectrum();
            System.out.println("nm: " + lastSpectrum[0]);
            System.out.println("spec: " + lastSpectrum[1]);
            System.out.println("temperature: " + pr655.getLastColorTemp());
        }
        System.out.println("DONE");
    }
}

File: single-language-analysis/dataset-files/python/test_smell_152.py
# Original URL: https://github.com/laurentperrinet/psychopy/blob/bc2411b431913aec4efb241c7fa1c08060681993/psychopy/tests/test_hardware/test_PR655.py#L8-L28

def test_smell_152():
    pr655 = hardware.findPhotometer(device='PR655')
    if pr655 is None:
        logging.warning('no device found')
    else:
        print(('type:', pr655.type))
        print(('SN:', pr655.getDeviceSN()))
        #on linux we do actually find a device that returns 'D'
        if pr655.type=='D':
            pytest.skip()
        pr655.measure()
        print(('lum', pr655.lastLum))
        print(('uv',pr655.lastUV))
        print(('xy', pr655.lastXY))
        print(('tristim', pr655.lastTristim))
        nm, spec = pr655.getLastSpectrum()
        print(('nm', nm))
        print(('spec', spec))
        print(('temperature', pr655.lastColorTemp))

    print('DONE')